connection.php

```diff
@@ -0,0 +1,725 @@
+    	/*if(empty($password) && empty($confirm_password))
+    	{
+    		echo "<script>alert('Enter your password... ')</script>";
+    		header('location:registration.php');
+    		exit;
+    	}
+	    else*/ if($password == $confirm_password)
+	    {
+
+	        $insert_employee = "INSERT INTO employee(first_name, middle_name, last_name, prefix, gender, dob, marital_status, employment, employer, email_id, password, image) VALUES('$first_name', '$middle_name', '$last_name', '$prefix', '$gender', '$dob', '$marital_status', '$employment', '$employer', '$email_id', '$password','$image[name]')" ;
+	    	 
+	        $result_employee = mysqli_query($this->connect,$insert_employee);
+	    	
+	        if($result_employee === TRUE)
+	        {
+	    	  $employee_id = mysqli_insert_id($this->connect);
+
+	    	  $insert_address = "INSERT INTO address(employee_id, type, street, city, state, mobile, landline, zip, fax)VALUES($employee_id,'residence','$home_street', '$home_city','$home_state', $home_mobile, $home_landline, $home_zip, $home_fax), ($employee_id,'office','$office_street', '$office_city','$office_state',     $office_mobile, $office_landline, $office_zip, $office_fax)";
+
+	    	  $result_address = mysqli_query($this->connect,$insert_address);
+
+	    	  $insert_communication = "INSERT INTO communication(employee_id, type) VALUES($employee_id, '$communication')";
+
+	    	  $result_communication = mysqli_query($this->connect,$insert_communication);
+
+	    	  mysqli_close($this->connect);
+	    	  echo "<script>alert('Registered Successfully..');</script>";
+	    	  header("location:login.php");
+	    	  exit;	
+	    	}
+	    	else
+	    	{
+	    	   echo "<script>alert('some error occured.....try again later')</script>";
+	    	   header("location:registration.php");
+	    	   exit;
+	        }
+    	}
+	    else
+	    {
+	  	    echo "<script>alert('incorrect password')</script>";
+	   	     exit;
+	    }
+   }
+
+	/*
+	 * @access: public
+	 * @param: file
+	 * @return:
+	*/
+	function image_upload($file)
+	{
+	 	$errors;
+	 	$file_name  = $file['name'];
+		$file_size  = $file['size'];
+		$file_tmp   = $file['tmp_name'];
+		$file_type  = $file['type'];
+		$file_ext   = strtolower(end(explode('.',$file['name'])));
+		$extensions = array("jpeg","jpg","png");
+
+		if(in_array($file_ext, $extensions) === false)
+		{
+	     	$errors = "extension not allowed, please choose a JPEG or PNG file.";
+		}
+
+		if (empty($errors))
```

Comment:
There is no check for the file size that is if file size is 0 or more than that system can handle.
what if someone select a image of 5GB size which can be a fake image. will system be able to handle it?